test/unit/node/generated-code-dependencies.js

```diff
@@ -347,6 +346,13 @@ suite('context test', function() {
     });
   });
 
+  function checkFile(outDir, filename, expected) {
```

Comment:
Yes, with a single long line the mocha diffs are hard to sort out.   I could split only if the lines differ if you prefer.